tracer/transport.go

```diff
@@ -22,6 +22,12 @@ type Transport interface {
 	SetHeader(key, value string)
 }
 
+// NewHTTPTransport returns a new Transport implementation using the given
+// delivery url.
+func NewHTTPTransport(url string) Transport {
```

Comment:
OK we could work on that. But we prefer not to expose the underlying transport mechanism because it's related to our agent integration. I'll keep you informed! Thank you for this precious feedback!